src/libcore/marker.rs

```diff
@@ -60,13 +76,16 @@ pub trait Sized {
 }
 
 /// Types that can be "unsized" to a dynamically sized type.
+///
+/// For example, the sized array type `[i8; 2]` implements `Unsize<[u8]>` and
```

Comment:
Yeah, that's something I don't understand particularly well. Do you have a reference?